languages/common-lisp/exercises/concept/conditionals/.docs/after.md

```diff
@@ -0,0 +1,164 @@
+## Summary
```

Comment:
This document currently does not contain link to external resources. Are there maybe resources to link to?

Reply:
A good catch! I couldn't find too many, but I'll transfer the ones from the hints document!